docs/content/getting-started/create-new-project.mdx

```diff
@@ -56,15 +56,6 @@ Here's a breakdown of the files and directories that are generated:
         <code>my_dagster_project.</code>.
```

Comment:
Just noticed this - there's an extra period at the end of the sentence. Mind taking care of it?